index.js

```diff
@@ -3,8 +3,15 @@
 const deepStrictEqual = require('deep-strict-equal')
 const rewind = require('@turf/rewind')
 
-const parseCoords = (s, transformCoords, stride = 2) => {
+const noTransform = (...coords) => coords
+
+const parseCoords = (s, opts = { transformCoords: noTransform, stride: 2 }, ctx = {} ) => {
```

Comment:
I think we should move the default values from `opts` to the top-most function: `parse`.

Reply:
Only `opts`? Not `ctx`? I don't have much experience with the new JS language features, don't have a good feeling yet - will be relying on your judgement.

Reply:
My personal opinion:

Functions like `parseCoords` should be considered "rather internal". Although they may be exported and hence subject to SemVer, their API should be optimised for internal use.

In this case defining the options closest to the user call (`parse`) makes most sense. Same for `ctx`.

Reply:
You would argue that given `parseCoords` a default/fallback value of `ctx` makes it more robust, but I'd rather not encourage inconsistent use of "rather internal" functions.

Reply:
I'll remove defaulting of `opts` and `ctx` from everywhere but `parse` .